lib/internal/util.js

```diff
@@ -213,6 +214,14 @@ function getConstructorOf(obj) {
   return null;
 }
 
+function getErrorName(code) {
+  if (errmap.has(code)) {
```

Comment:
should `code` be type checked at all?

Reply:
@jasnell Yeah, `errname()` does `CHECK_LT(err, 0)`, we should probably check for the type and the ranges. Thanks for catching that.